replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/Syncer.java

```diff
@@ -177,7 +177,10 @@ SyncResponse sync() {
           }
         }
 
-        replicationStatus.setLastMetadataModified(metadata.getMetadataModified());
+        if (replicationStatus.getLastMetadataModified() == null
+            || metadata.getMetadataModified().after(replicationStatus.getLastMetadataModified())) {
```

Comment:
❓ Is this the case where we are only transferring previously failed items? Can we add a test for this as well if it does not exist.

Reply:
Yes. Added a test.